src/components/ui/Breadcrumb/breadcrumb.scss

```diff
@@ -41,21 +66,89 @@
 
     [data-breadcrumb-divider] {
       height: var(--fs-spacing-3);
-      margin: 0 var(--fs-spacing-2) 0 var(--fs-spacing-1);
+      margin: var(--fs-spacing-1);
       border-left: var(--fs-border-width-default) solid var(--fs-border-color-light);
     }
 
     [data-store-link] {
-      display: inline;
       padding: 0;
 
       &:visited {
         color: var(--fs-color-link);
       }
     }
 
+    [data-store-dropdown-button] {
+      display: inline;
+      padding: 0 calc(var(--fs-spacing-0) / 2);
+      margin: calc(var(--fs-spacing-0) / 4);
+      color: var(--fs-color-link);
+      letter-spacing: var(--fs-spacing-0);
+      cursor: pointer;
+      background-color: transparent;
+      border-width: 0;
+      border-radius: var(--fs-spacing-0);
+
+      &:focus, &:focus-visible {
+        @include focus-ring;
+      }
+
+      span {
+        display: inline-block;
+        font-size: var(--fs-text-size-3);
+        font-weight: var(--fs-text-weight-bold);
+        transform: translate(rem(1px), rem(-6px));
+      }
+    }
+
     [data-breadcrumb-item="current"] {
       color: var(--fs-color-text-light);
     }
   }
 }
+
+[data-store-dropdown-menu] {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  max-width: 50%;
+  margin-top: var(--fs-spacing-0);
+  overflow: hidden;
+  background: var(--fs-color-tertiary-bkg);
+  border-radius: var(--fs-border-radius-default);
+  box-shadow: var(--fs-shadow);
+
+  [data-store-dropdown-item] {
+    display: flex;
+    align-items: center;
+    width: 100%;
+    padding: var(--fs-spacing-1) var(--fs-spacing-2);
+    font-size: var(--fs-text-size-base);
+    font-weight: var(--fs-text-weight-regular);
+    color: var(--fs-color-link);
+    cursor: pointer;
+    background-color: transparent;
+    border-width: 0;
+
+    @include media(">=notebook") {
+      &:focus, &:hover {
+        text-decoration: underline;
```

Comment:
This focus looks cropped
![image](https://user-images.githubusercontent.com/11613011/161570053-e1f8bf0c-e4f9-49e8-8a9f-0e1026ade0dc.png)

Reply:
I updated [the Preview](https://preview-446--base.preview.vtex.app/aedle-vk1-headphone-99988211/p) with this problem solved

![Screen Shot 2022-04-04 at 14 51 20](https://user-images.githubusercontent.com/51174217/161602366-820fb867-0e88-4cda-8921-73992035d39e.png)